compiler/passes/scopeResolve.cpp

```diff
@@ -1922,8 +1922,8 @@ static void resolveEnumeratedTypes() {
         if (EnumType* type = toEnumType(first->var->type)) {
           if (SymExpr* second = toSymExpr(call->get(2))) {
             const char* name;
-            bool found = false;
-
+            //bool found = false;
+            /*
```

Comment:
Huh.  We work just fine for tuple.size calls.  There's probably something deeper going wrong.